packages/definitions-parser/src/lib/definition-parser.ts

```diff
@@ -773,9 +793,12 @@ function checkAllUsedRecur(ls: Iterable<string>, usedFiles: Set<string>, unusedF
 
   for (const unusedFile of unusedFiles) {
     if (usedFiles.has(unusedFile)) {
-      throw new Error(
-        `File ${fs.debugPath()}/${unusedFile} listed in ${unusedFilesName} is already reachable from tsconfig.json.`
-      );
+      // TODO: reenable after fixing up DT
```

Comment:
what's needed to do that? I hope that we are missing lots of files from just a few projects, since there are only 113 OTHER_FILES.txt to begin with. But I guess it might be a few files from up to 113 projects.